tests/App.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import App from '../client/src/app';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
If you end up creating a bunch of individual test files for various components, it can be helpful to put the Enzyme config in a separate testSetup.js file that Jest points to. That way you can run Jest tests on the individual component test file while you're developing it, rather than running the app's entire test suite every time :+1: